picobox/_box.py

```diff
@@ -93,6 +150,20 @@ def get(self, key, default=_missing):
         return value
 
     def pass_(self, key, as_=_missing):
+        """Pass a dependency to a function if nothing explicitly passed.
+
+        The decorator implements late binding which means it does not require
+        to have a dependency instance in the box before applying. The instance
+        will be looked up when a decorated function is called. Other imporant
+        thins is that it doesn't change a signature of decorated function
```

Comment:
Other important "thing" is misspelled. Could be replaced with "Other important property...".